translations/en.json

```diff
@@ -509,9 +509,9 @@
   "record.metadata.quality.updateFrequency.failed": "Update frequency is not specified",
   "record.metadata.quality.updateFrequency.success": "Update frequency is specified",
   "record.metadata.related": "Explore the catalog",
-  "record.metadata.resourceCreated": "Created on {date}",
-  "record.metadata.resourcePublished": "Published on {date}",
-  "record.metadata.resourceUpdated": "Last updated on {date}",
+  "record.metadata.resourceCreated": "Created ",
```

Comment:
I'm wondering if we shouldn't rather handle the space (when the directive is used in the same line) within the html/TS. I guess it would be hard to handle this within the directive itself though, as it can be used inline and in new lines!?